src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAXIMUM_POSSIBLE_NUMBER = 100;
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
We can add field with type `ColorSupplier` to our class, so we won't need to create an instance every time when method invoked